### PR TITLE
Add memcached config to docker-compose.yml

### PR DESCRIPTION
**Pull request by andrew**
I think this is correct, not done much docker composing before though so would be good to get a second opinion

### CODE REVIEW

**Comment by tarebyte** [comment]
@andrew won't we have to update the `config/environments/development.rb` file in order for this to be used?

**Comment by andrew** [comment]
D'oh! Yeah good point @tarebyte, I forget that the docker is only for development atm

File: docker-compose.yml [hunk 1]
```diff
@@ -23,4 +25,5 @@ services:
       - OCTOBOX_DATABASE_USER=postgres
       - OCTOBOX_DATABASE_PASSWORD=development
       - OCTOBOX_DATABASE_HOST=database.service.octobox.internal
+      - MEMCACHEDCLOUD_SERVERS=memcached.service.octobox.internal
```
Comment (tarebyte) [inline docker-compose.yml]: If we want to mirror a "production" environment in development, we may also need to set the username and password for the memcached service ([file](https://github.com/octobox/octobox/blob/master/config/environments/production.rb#L56-L59))

Comment (andrew) [inline docker-compose.yml]: Agreed, would be good to make the docker setup usable for dev, test and production

Comment (andrew) [inline docker-compose.yml]: Turns out standard memcached doesn't have any username/password support built in, memcachedcloud adds that on top of it.

**Pull request merged**